datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -23,6 +24,34 @@ def __is_canonical(version):
     return re.match(P, version) is not None
 
 
+def __find_shipped_integrations():
+    root = os.path.dirname(os.path.abspath(__file__))
+    filename = 'requirements-agent-release.txt'
+
+    integrations = set()
+
+    while True:
+        file_path = os.path.join(root, filename)
+        if os.path.isfile(file_path):
+            break
+
+        new_root = os.path.dirname(root)
+        if new_root == root:
+            return integrations
+
+        root = new_root
+
+    with open(file_path, 'rb') as f:
+        contents = f.read().decode('utf-8')
+
+    for line in contents.splitlines():
+        integration, separator, _ = line.strip().partition('==')
+        if separator:
+            integrations.add(integration)
```

Comment:
Is there a case where `if separator` is needed ? It seems there is always `==` for each line.

Reply:
Just in case

Reply:
Silently skipping is not great though. We should at least output a warning/error IMO.

Reply:
There might eventually be a comment, blank line, etc.